go.mod

```diff
@@ -78,18 +79,18 @@ require (
 	go.uber.org/zap v1.19.0
 	golang.org/x/crypto v0.1.0
 	golang.org/x/net v0.4.0
-	golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b
+	golang.org/x/oauth2 v0.0.0-20220822191816-0ebed06d0094
 	golang.org/x/sync v0.1.0
 	golang.org/x/sys v0.3.0
 	golang.org/x/term v0.3.0
 	golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
 	golang.org/x/tools v0.4.0
-	google.golang.org/api v0.60.0
-	google.golang.org/genproto v0.0.0-20220502173005-c8bf987b8c21
+	google.golang.org/api v0.84.0
+	google.golang.org/genproto v0.0.0-20220616135557-88e70c0c3a90
 	google.golang.org/grpc v1.51.0
 	google.golang.org/protobuf v1.28.1
 	gopkg.in/gcfg.v1 v1.2.0
-	gopkg.in/square/go-jose.v2 v2.2.2
+	gopkg.in/square/go-jose.v2 v2.6.0
```

Comment:
I'm surprised this dependency broke the signature of NewNumericDate on this bump

Reply:
Yes, it’s a breaking change in 2.3.0 which doesn’t even get a mention in the release notes :-/.

Reply:
There’s a forthcoming bump to v3 in go-oidc, https://github.com/coreos/go-oidc/pull/360; perhaps this should wait for a corresponding release (and bump the dependency across all of k8s/k8s).

Reply:
I’ve split out the go-jose changes into a separate commit, and updated go-oidc to v3.5.0.